packages/components/src/radio-control/index.tsx

```diff
@@ -88,7 +89,13 @@ export function RadioControl(
 							onChange={ onChangeValue }
 							checked={ option.value === selected }
 							aria-describedby={
-								!! help ? `${ id }__help` : undefined
+								// TODO: will improve if we like this solution
+								// eslint-disable-next-line no-nested-ternary
```

Comment:
If there are any ways to avoid the nested ternaries, let's do it, including if it requires us to change the `map` return syntax. Nested ternaries are discouraged for a good reason - readability and clarity can be diminished.

Reply:
Yup, I just didn't invest time refactoring the nested ternary because I wanted to make sure the current approach was going in the right direction (see https://github.com/WordPress/gutenberg/pull/63751#discussion_r1684375290). The nested ternary will be gone in any case before final code review.